desktop/SmesherManager.ts

```diff
@@ -27,13 +28,23 @@ const logger = Logger({ className: 'SmesherService' });
 class SmesherManager extends AbstractManager {
   private smesherService: SmesherService;
 
+  private smesherMetadataService: SmesherMetadataService;
+
   private readonly configFilePath: string;
 
-  constructor(mainWindow: BrowserWindow, configFilePath: string) {
+  private genesisID: string;
+
+  constructor(
+    mainWindow: BrowserWindow,
+    configFilePath: string,
+    genesisID: string
+  ) {
     super(mainWindow);
     this.smesherService = new SmesherService();
     this.smesherService.createService();
     this.configFilePath = configFilePath;
+    this.genesisID = genesisID;
```

Comment:
Double-check it accurately so that nothing will break in case of switching networks...

p.s.
That's why I don't like to store some values (that can be calculated or should be passed from a "higher" level) in internal states of classes or components. :)
Another pretty robust approach might be injecting dependency that will hold actual `genesisID` (or calculate it). Then we may be sure that every class (NodeManager, SmesherManaher, etc) has the same value...
But it will require some refactoring, not sure we have enough time for that now.